site/src/Form/Type/NotificationSubscriptionType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class NotificationSubscriptionType extends AbstractType
+{
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array                $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add('subscribedToEmailNotifications', ChoiceType::class, [
```

Comment:
How about making this a simple checkbox?

[ ] Receive notifications via e-mail